src/components/auth/torus/sdk/strategies.js

```diff
@@ -10,13 +8,19 @@ export const LoginStrategy = {
   PaswordlessSMS: 'auth0-pwdless-sms',
 
   getTitle(strategy) {
-    const { GoogleLegacy, PaswordlessSMS } = LoginStrategy
-
-    if (strategy.includes('pwdless')) {
-      return `Passwordless (${strategy === PaswordlessSMS ? 'SMS' : 'E-Mail'})`
+    switch (strategy) {
```

Comment:
my implementation performs the same, but with a less of code
you need to add just

```javascript
if (!strategy) {
  return 'Self Custody'
}
```
to the beginning